### PR TITLE
Portfolio desktop version with media queries

### PR DESCRIPTION
**Pull request by jash-debug**
Add classes and text to the html structure
Add media queries at the bottom of the CSS for the desktop version
Add images 
Fix linter errors
Add static header
Add images and classes for responsive design
Fix errors in as directed in review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -273,14 +327,16 @@ <h1 class="form__h1 poppins">
...
+                            placeholder="email@something.com" required />
                     </li>
                     <li class="form__list2">
-                        <textarea class="poppins form__list__style" name="textarea__form" id="textarea__form" cols="30" rows="10" maxlength="500"
-                            placeholder="Write your message here" required></textarea>
+                        <textarea class="poppins form__list__style" name="textarea__form" id="textarea__form" cols="30"
+                            rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
                     </li>
                     <li>
                         <button type="submit" class="form__button poppins">Get In Touch</button>
```
Comment (leonardodiasb) [inline index.html:342]: - [x] The Get In Touch button of the Figma template on the desktop version of the form is aligned in the center. Kindly reposition this button so the form can match with the Figma template.
 -Your design:
image
-Figma template:
image

File: index.html [hunk 2]
```diff
@@ -51,212 +56,261 @@ <h1 class="hero__heading poppins">
...
-                    see Project
-                </button>
+                <div>
+                    <button type='button' class="project__button poppins">
+                        see Project
+                    </button>
+                </div>
             </div>
 
         </div>
```
Comment (leonardodiasb) [inline index.html:135]: - [x] Some project cards are not responsive and their images do not resize. This is causing the content of the project cards to be displayed outside of the card's boundaries and out of the screen on screen sizes close to the 768px transition. This is a bad visual design. Please review the sizing of elements inside the card so they can be responsive. 
 -Your design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @jash-debug, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @jash-debug, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**